qa/tasks/ceph_test_case.py

```diff
@@ -52,7 +52,7 @@ def get_ceph_cmd_stdout(self, *args, **kwargs):
                 args = args[0]
             kwargs['args'] = args
         kwargs['stdout'] = kwargs.pop('stdout', StringIO())
-        return self.run_ceph_cmd(**kwargs).stdout.getvalue()
```

Comment:
BTW, do we have any use case for this improvement ?

Reply:
This should be a minor enhancement when it comes log checking the return string or any logging used by tests.

Reply:
Okay, if so maybe just copy the log segment in this PR could help to understand it.